### PR TITLE
Update the TEE version to 14.03 and the copyright year to 2016.

### PR DESCRIPTION
**Pull request by AlexRukhlin**

### CODE REVIEW

File: source/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 1]
```diff
@@ -470,7 +470,7 @@
 						<b>Integrated development environment (IDE) requirements:</b>
 					</p>
 					<ul xmlns="">
-						<li>IDEs that are based on Eclipse 3.5 to 4.4 on Windows, Linux, Mac OS X, Solaris, AIX, or HP-UX.</li>
+						<li>IDEs that are based on Eclipse 3.5 to 4.5 on Windows, Linux, Mac OS X, Solaris, AIX, or HP-UX.</li>
```
Comment (jpricketMSFT) [inline source/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: Is this the place to put the warning:
This is the last version of TEE that will support Eclipse 3.5. Future versions will only support 4.? and higher.
?

**Comment by DavidStaheli** [comment]
It looks like some of the localized (i.e. Japanese) Unicode strings were corrupted.

File: translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 2]
```diff
@@ -436,53 +436,53 @@
     }
 </STYLE>
 </head>
-<body><input type="hidden" id="userDataCache" class="userDataStyle"></input><input type="hidden" id="hiddenScrollOffset"></input><img id="dropDownImage" style="display:none; height:0; width:0;" src="drpdown.gif"><img id="dropDownHoverImage" style="display:none; height:0; width:0;" src="drpdown_orange.gif"><img id="collapseImage" style="display:none; height:0; width:0;" src="collapse.gif"><img id="expandImage" style="display:none; height:0; width:0;" src="exp.gif"><img id="copyImage" style="display:none; height:0; width:0;" src="copycode.gif"><img id="copyHoverImage" style="display:none; height:0; width:0;" src="copycodeHighlight.gif"><div id="header"><table width="100%" id="topTable"><tr id="headerTableRow1"><td align="left"><span id="runningHeaderText"></span></td></tr><tr id="headerTableRow2"><td align="left"><span id="nsrTitle">Team Explorer Everywhere プラグインとコマンドライン クライアントのインストール</span></td></tr><tr id="headerTableRow3"></tr></table></div><div id="mainSection"><div id="mainBody"><div id="allHistory" class="saveHistory"></div>
```
Comment (jpricketMSFT) [inline translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: The changes in this file don't seem to be desirable. You may need to use a hex editor to change just the numbers or get the correct Fonts installed.

File: translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<html DIR="LTR" xmlns:MSHelp="http://msdn.microsoft.com/mshelp" xmlns:ddue="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:tool="http://www.microsoft.com/tooltip"><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; CHARSET=utf-8"></META><META NAME="save" CONTENT="history"></META><title>Team Explorer Everywhere プラグインとコマンドライン クライアントの言語パックのインストール</title><STYLE TYPE="text/css">
```
Comment (jpricketMSFT) [inline translations/ja/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: Another case of not having the right font loaded.

File: translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<html DIR="LTR" xmlns:MSHelp="http://msdn.microsoft.com/mshelp" xmlns:ddue="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:tool="http://www.microsoft.com/tooltip"><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; CHARSET=UTF-8"></META><META NAME="save" CONTENT="history"></META><title>正在安装 Team Explorer Everywhere 插件和命令行客户端</title><STYLE TYPE="text/css">
```
Comment (jpricketMSFT) [inline translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: More problems with the language being converted unintentionally.

File: translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 5]
```diff
@@ -1,4 +1,4 @@
-<html DIR="LTR" xmlns:MSHelp="http://msdn.microsoft.com/mshelp" xmlns:ddue="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:tool="http://www.microsoft.com/tooltip"><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; CHARSET=utf-8"></META><META NAME="save" CONTENT="history"></META><title>正在安装 Team Explorer Everywhere 插件和命令行客户端语言包</title><STYLE TYPE="text/css">
```
Comment (jpricketMSFT) [inline translations/zh_CN/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: same problem.

File: translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<html DIR="LTR" xmlns:MSHelp="http://msdn.microsoft.com/mshelp" xmlns:ddue="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:tool="http://www.microsoft.com/tooltip"><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; CHARSET=UTF-8"></META><META NAME="save" CONTENT="history"></META><title>正在安裝 Team Explorer Everywhere 外掛程式與命令列用戶端</title><STYLE TYPE="text/css">
```
Comment (jpricketMSFT) [inline translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE.htm]: same problem.

File: translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm [hunk 7]
```diff
@@ -1,4 +1,4 @@
-<html DIR="LTR" xmlns:MSHelp="http://msdn.microsoft.com/mshelp" xmlns:ddue="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:tool="http://www.microsoft.com/tooltip"><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; CHARSET=UTF-8"></META><META NAME="save" CONTENT="history"></META><title>正在安裝 Team Explorer Everywhere 外掛程式與命令列用戶端語言套件</title><STYLE TYPE="text/css">
+<html DIR="LTR" xmlns:MSHelp="http://msdn.microsoft.com/mshelp" xmlns:ddue="http://ddue.schemas.microsoft.com/authoring/2003/5" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:tool="http://www.microsoft.com/tooltip"><head><META HTTP-EQUIV="Content-Type" CONTENT="text/html; CHARSET=UTF-8"></META><META NAME="save" CONTENT="history"></META><title>正在安�? Team Explorer Everywhere 外掛程�?與命令列用戶端語言套件</title><STYLE TYPE="text/css">
```
Comment (jpricketMSFT) [inline translations/zh_TW/com.microsoft.tfs.client.eclipse.help/content/InstallTEE_LP.htm]: same problem.

**Comment by jpricketMSFT** [comment]
Don't check in the JA or ZH files. They have bad changes.
The rest of the files look good.

**Comment by AlexRukhlin** [comment]
I've restored all the translated resources. It looks like we didn't touch them in the previous version upgrades. That was not 100% clear from the version update utility instructions. I'll add a note to the latter.

**Pull request merged**